mosquitto/docker-compose.yml

```diff
@@ -2,12 +2,9 @@
 version: "3"
 services:
   mosquitto:
-    image: hub.foundries.io/<your factory>/mosquitto:latest
-    tmpfs:
-      - /run
-      - /var/lock
-      - /var/log
+    image: eclipse-mosquitto:1.6.12
+    volumes:
+      - /var/run/secrets/mosquitto.conf:/mosquitto/config/mosquitto.conf
```

Comment:
I guess you assume a user is going to run `fioctl device config ...`  *before* they install this app? That seems like a dangerous assumption. Will Mosquitto start if the file doesn't exist? I think you had a reason for not including a default mosquitto.conf in this sample? Can't remember but doing that would really simplify this.

Reply:
I will add these steps in the tutorial, basically I left the config file there as a reference. 
Answering your questions: No the container will not work without the file at "/var/run/secrets/mosquitto.conf".
It is necessary to use fioctl to send the config file to the device.
I can think on two suggestions:
 - Assume that everyone how will try this app will read the document: https://docs.foundries.io/latest/tutorials/mosquitto.html
And maybe we should add a readme file referring to the tutorial in the mosquitto folder.
- Add just a readme file with the node explaining that you need to send the file over fioctl.